spec/support/drivers_path_helpers.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+ENV['DRIVERS_PATH'] ||= 'drivers'
+
+module DriversPathHelpers
+  def self.included(klass)
+    klass.send(:extend, DriversPathHelpers)
+
+    RSpec::Matchers.define :have_driver_file do |file_path|
+      match do |dummy_app_path|
+        File.exist? File.expand_path(File.join(dummy_app_path, drivers_path, file_path))
+      end
+    end
+  end
+
+  def drivers_path
+    ENV['DRIVERS_PATH']
+  end
```

Comment:
Instead of modifying the environment on line 3, how about we use an "or" here?

```suggestion
  def drivers_path
    ENV['DRIVERS_PATH'] || RailsDrivers::Config::DEFAULT_DRIVERS_PATH
  end
```

Reply:
ah now I remember why I did that... in this support file we cannot load `RailsDrivers` yet because of dependencies on rails (which also hasn't been loaded by this point)

Reply:
Ah, gotcha. Then perhaps just a literal `|| 'drivers'` is fine then, yeah?